packages/playground/blueprints/src/lib/steps/common.ts

```diff
@@ -1,7 +1,8 @@
 import type { PHPResponse, UniversalPHP } from '@php-wasm/universal';
+import { JSDOM } from 'jsdom';
 
 export function asDOM(response: PHPResponse) {
-	return new DOMParser().parseFromString(response.text, 'text/html')!;
+	return new JSDOM(response.text).window.document;
```

Comment:
@adamziel, the current `activatePlugin` calls this function, and in NodeJS  `DOMParser` doesn't exist.
Is it ok if I install/use [`jsdom`](https://www.npmjs.com/package/jsdom) library to be used in `blueprints` package?

Reply:
I also need to set `jsdom` as external in `playground-client`
I'm happy to refactor activatePlugin to parse it in `wp-now` and then call the function that activates it.

Reply:
@adamziel , done here https://github.com/WordPress/wordpress-playground/pull/351/commits/ed28a2be646445f436936510b08206ace597376e
I had to change all the calls to be async/await.